keyboards/ergodash/rev1/keymaps/rmarinbe/keymap.c

```diff
@@ -0,0 +1,140 @@
+        [1] = LAYOUT_4key(KC_F11, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, TO(0), KC_NLCK, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F12, KC_MINS, KC_EQL, KC_COMM, KC_LBRC, KC_RBRC, KC_COLN, KC_AMPR, KC_PERC, KC_ASTR, KC_P7, KC_P8, KC_P9, KC_PLUS, KC_GT, KC_CAPS, KC_QUOT, KC_DQUO, KC_LPRN, KC_RPRN, KC_SCLN, KC_PIPE, KC_CALC, KC_SLSH, KC_P4, KC_P5, KC_P6, KC_MINS, KC_LT, KC_LSFT, KC_EXLM, KC_AT, KC_HASH, KC_DLR, KC_PERC, KC_LALT, KC_RALT, KC_P0, KC_P1, KC_P2, KC_P3, KC_EQL, KC_CIRC, KC_BSPC, TO(0), TO(0), KC_NO, KC_SPC, KC_TAB, KC_NO, KC_NO, KC_BSPC, KC_ENT, KC_P0, KC_PDOT, TO(0), KC_BSPC),
+        [2] = LAYOUT_4key(KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_QUES, KC_PIPE, KC_NO, KC_NO, KC_NO, KC_LBRC, KC_RBRC, KC_NO, KC_CAPS, KC_EXLM, KC_AT, KC_HASH, KC_DLR, KC_PERC, KC_SLSH, KC_BSLS, KC_CIRC, KC_AMPR, KC_ASTR, KC_LPRN, KC_RPRN, KC_UNDS, KC_EQL, KC_1, KC_2, KC_3, KC_4, KC_5, KC_NO, KC_NO, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_ESC, TO(0), TO(0), KC_NO, KC_SPC, KC_TAB, KC_NO, KC_NO, KC_BSPC, KC_ENT, KC_NO, TO(0), TO(0), KC_BSPC),
+        [3] = LAYOUT_4key(KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, TO(0), KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, RESET, RGB_TOG, RGB_MOD, RGB_HUD, RGB_HUI, KC_NO, KC_NO, RGB_SAD, RGB_SAI, RGB_VAD, RGB_VAI, KC_NO, KC_NO, KC_NO, KC_NO, BL_TOGG, BL_BRTG, BL_INC, BL_DEC, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO),
+        [4] = LAYOUT_4key(KC_NO, KC_NO, KC_ACL0, KC_ACL1, KC_ACL2, KC_NO, KC_MUTE, TO(0), KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_SLEP, KC_NO, KC_NO, KC_BTN2, KC_MS_U, KC_BTN1, KC_NO, KC_VOLU, KC_MPLY, KC_MRWD, KC_NO, KC_NO, KC_MFFD, KC_NO, KC_NO, KC_NO, KC_WH_L, KC_MS_L, KC_MS_D, KC_MS_R, KC_WH_R, KC_VOLD, KC_MSTP, KC_LEFT, KC_DOWN, KC_UP, KC_RGHT, KC_NO, KC_NO, KC_NO, KC_NO, KC_WH_D, KC_BTN3, KC_WH_U, KC_NO, KC_NO, KC_NO, KC_MPRV, KC_NO, KC_NO, KC_MNXT, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO)
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwerty[][2]     = SONG(QWERTY_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Reply:
done